File: 3_het/3het1.py
import numpy as np

def average():
    szam = 0.0
    ossz = 0.0
    atlag = 0.0
    print("Irjon be 10 szamot: ")
    for i in range(0,10):
        szam = float((input(f"{i+1}. szam: ")))
        ossz = ossz+szam
    atlag = ossz/10
    print(f"A beirt szamok atlaga: {atlag}")

average()

File: 3_het/3het1.cpp
#include <iostream>
using namespace std;

int main() {
    int szam;
    double ossz;
    cout << "Irjon be 10 szamot: ";
    for (int i = 0; i < 10; i++)
    {
        cin >> szam;
        ossz += szam;
    }
    cout << "A beirt szamok atlaga: " << ossz/10 << endl;
    return 0;
}